bee-common/bee-packable/tests/boxed_slice_prefix.rs

```diff
@@ -18,18 +18,15 @@ fn boxed_slice_prefix_from_boxed_slice_invalid_error() {
     let boxed_slice = vec![0u8; 16].into_boxed_slice();
     let prefixed = BoxedSlicePrefix::<u8, BoundedU32<1, 8>>::try_from(boxed_slice);
 
-    assert!(matches!(
-        prefixed,
-        Err(TryIntoPrefixError::Invalid(InvalidBoundedU32(16)))
-    ));
+    assert!(matches!(prefixed, Err(TryIntoBoundedU32Error::Invalid(16))));
 }
 
 #[test]
 fn boxed_slice_prefix_from_boxed_slice_truncated_error() {
     let boxed_slice = vec![0u8; 257].into_boxed_slice();
     let prefixed = BoxedSlicePrefix::<u8, u8>::try_from(boxed_slice);
 
-    assert!(matches!(prefixed, Err(TryIntoPrefixError::Truncated(257))));
+    assert!(prefixed.is_err());
```

Comment:
Why are we being less specific now ?

Reply:
because`<u8 as TryFrom<usize>>::Error` is  [`TryFromIntError`](https://doc.rust-lang.org/std/num/struct.TryFromIntError.html) and it's only field is private, so we cannot `matches!` it.

Reply:
Ok